test/unit/carousel.ts

```diff
@@ -0,0 +1,414 @@
+import { utils } from '@storefront/core';
+import * as sinon from 'sinon';
+import Carousel from '../../src/carousel';
+import suite from './_suite';
+
+suite('Carousel', ({ expect, spy, stub }) => {
+
+  describe('settings', () => {
+    it('should default to default settings if no settings are passed in', () => {
+      const carousel = new Carousel();
+
+      expect(carousel.props.speed).to.eql(0);
```

Comment:
If you're just comparing simple types (eg ones where === suffices, then use `eq`)